src/api/app/controllers/webui/comments_controller.rb

```diff
@@ -12,4 +30,15 @@ def destroy
     end
     redirect_back(fallback_location: root_path)
   end
+
+  private
+
+  def permitted_params
+    params.require(:comment).permit(:body, :parent_id)
+  end
+
+  def find_commented
+    commentable = [Project, Package, BsRequest].find { |klass| klass.name == params[:commentable_type] }
```

Comment:
I would call this `class` as `commentable` is the name of the relation, so for me it makes more sense to call this `class` and `commentable` to what you have called `commented`


 and you can use 

``` Ruby
class = params[:commentable_type].constantize
```